### PR TITLE
docs: Update link to Karma homepage.

### PR DESCRIPTION
**Pull request by robphoenix**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: aio/content/guide/testing.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Just run this one CLI command:
 </code-example>
 
 The `ng test` command builds the app in _watch mode_,
-and launches the [karma test runner](https://karma-runner.github.io/1.0/index.html).
+and launches the [karma test runner](https://karma-runner.github.io/3.0/index.html).
```
Comment (gkalpak) [inline aio/content/guide/testing.md]: Ideally, we should be using just https://karma-runner.github.io/.

But currently it redirects to the v2.x docs, with a notification saying this version is old and a link to the v3.x docs.
Could you open an issue on the Karma repo, @robphoenix?

Comment (robphoenix) [reply aio/content/guide/testing.md]: Sure, done - https://github.com/karma-runner/karma/issues/3189

Comment (gkalpak) [reply aio/content/guide/testing.md]: Thx! Let's wait for a response from the Karma team.

Comment (robphoenix) [reply aio/content/guide/testing.md]: The Karma team have updated their homepage :tada: I've update the PR.

File: aio/content/guide/testing.md [hunk 2]
```diff
@@ -188,7 +188,7 @@ You'll need to push a new commit to trigger a build.
 
 When the CLI commands `ng test` and `ng e2e` are generally running the CI tests in your environment, you might still need to adjust your configuration to run the Chrome browser tests.
 
-There are configuration files for both the [Karma JavaScript test runner](http://karma-runner.github.io/2.0/config/configuration-file.html) 
+There are configuration files for both the [Karma JavaScript test runner](http://karma-runner.github.io/latest/config/configuration-file.html)
```
Comment (gkalpak) [inline aio/content/guide/testing.md]: While you're at it, could change the links to `https` as well? 🙏

**Review by gkalpak — APPROVED** [review]

File: aio/content/guide/testing.md (hunk 2 above)
Comment (robphoenix) [reply aio/content/guide/testing.md]: :smile: sure

Comment (robphoenix) [reply aio/content/guide/testing.md]: there's a couple that aren't yet https so I've left them as is, but updated all the other links to https where possible. I've also fixed my previous commit message.

**Review by gkalpak — APPROVED** [review]

**Comment by gkalpak** [comment]
Now all you need to do is change the commit messages to `docs: ...` and this will be good to go :grin:
Thx for following through :smiley:

**Comment by gkalpak** [comment]
No, `testing` is not a valid scope (see [commit message guidelines][1]).
Use `docs: some message here` instead.

[1]: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit

**Comment by robphoenix** [comment]
ah, sorry. OK, updated, I *think* we're almost there.... :smile:

**Comment by gkalpak** [comment]
Awesome! It just needs approval from the `testing` group owners.
cc @IgorMinar, @jenniferfell

**Comment by robphoenix** [comment]
Grand, thanks for your patience @gkalpak :+1:

**Review by IgorMinar — APPROVED** [review]

**Pull request closed without merging**